src/components/grid/grid.test.js

```diff
@@ -451,36 +343,38 @@ context("Testing Grid component", () => {
               },
             ]}
           >
-            <pod alignTitle="left" border padding="medium" variant="primary">
-              3
-            </pod>
+            <Pod>Item 3</Pod>
           </GridItem>
         </GridContainer>
       );
 
-      gridComponent()
-        .children()
+      gridItem(0)
         .should("have.css", "grid-column", "1 / 7")
         .and("have.css", "grid-row", "1 / 1")
-        .and("have.css", "width", "623px")
         .and("have.css", "align-self", "stretch")
         .and("have.css", "justify-self", "stretch");
-      gridComponent()
-        .children()
-        .eq(1)
+      gridItem(0)
+        .invoke("css", "width")
+        .then(parseFloat)
+        .then((width) => expect(width).to.be.within(623, 624));
+      gridItem(1)
         .should("have.css", "grid-column", "6 / 13")
         .and("have.css", "grid-row", "1 / 1")
-        .and("have.css", "width", "8px")
         .and("have.css", "align-self", "end")
         .and("have.css", "justify-self", "end");
-      gridComponent()
-        .children()
-        .eq(2)
+      gridItem(1)
+        .invoke("css", "width")
+        .then(parseFloat)
+        .then((width) => expect(width).to.be.within(75, 76));
+      gridItem(2)
         .should("have.css", "grid-column", "1 / 13")
         .and("have.css", "grid-row", "3 / 3")
-        .and("have.css", "width", "1270px")
         .and("have.css", "align-self", "start")
         .and("have.css", "justify-self", "stretch");
+      gridItem(2)
```

Comment:
I would rather not use `invoke` here, just use the `jQuery` method using 

`gridItem(2).then(($el) => {
     expect($el.css("width"))...
})`